managed/services/management/node.go

```diff
@@ -34,27 +34,8 @@ import (
 	"github.com/percona/pmm/managed/services"
 )
 
-// NodeService represents service for working with nodes.
-type NodeService struct {
-	db  *reform.DB
-	akp apiKeyProvider
-
-	l *logrus.Entry
-
-	managementv1.UnimplementedNodeServiceServer
-}
-
-// NewNodeService creates NodeService instance.
-func NewNodeService(db *reform.DB, akp apiKeyProvider) *NodeService {
-	return &NodeService{
-		db:  db,
-		akp: akp,
-		l:   logrus.WithField("component", "node"),
-	}
-}
-
 // RegisterNode performs the registration of a new node.
-func (s *NodeService) RegisterNode(ctx context.Context, req *managementv1.RegisterNodeRequest) (*managementv1.RegisterNodeResponse, error) {
+func (s *ServiceService) RegisterNode(ctx context.Context, req *managementv1.RegisterNodeRequest) (*managementv1.RegisterNodeResponse, error) {
```

Comment:
it's not really `Service` related method :)

Reply:
Right, it doesn't look semantic now :( 

I believe renaming it to ManagementService will be much better, WDYT?

Reply:
Problem is that we have way too many services, which is why we end up running out of good names. Example: MgmtService, which is also `ManagementService v1beta1`, so the only difference between them is the version, i.e. `v1` versus `v1beta1`.

IMO this transition will necessitate making a few compromises or workarounds anyway :)

Reply:
I think we need to differentiate them by using namings which makes it clear that it's beta like `ManagementService` and `ManagementServiceV1Beta`

Reply:
OK, good suggestion! Will do.

Reply:
Done!

<img width="1440" alt="Screenshot 2024-02-27 at 11 48 49" src="https://github.com/percona/pmm/assets/81549/8f76ca24-9a15-42f6-8f91-861d77e9b025">